datadog/Tiltfile

```diff
@@ -105,22 +132,35 @@ def datadog_keys_secret():
     )
 
 
-def helm_resources():
+def helm_resources(agent_image_tag = None):
+    """Create Tilt resources required to install Datadog Agent from Helm charts.
+
+    Args:
+      agent_image_tag: Sets a specific image tag version for the Datadog Agent.
+    """
     helm_repo(
         "helm-datadog-repo",
         "https://helm.datadoghq.com",
         labels = ["datadog"],
     )
 
+    flags = []
+
     chart_values = os.path.join(extension_path, "values.yaml")
+    flags.append(["--values", chart_values])
+
+    if agent_image_tag:
+        flags.append(["--set", "agents.image.tag=%s" % agent_image_tag])
+        flags.append(["--set", "clusterAgent.image.tag=%s" % agent_image_tag])
+        flags.append(["--set", "clusterChecksRunner.image.tag=%s" % agent_image_tag])
+    else:
+        agent_image_tag = "latest"
+
     helm_resource(
         workload_name,
         chart = "helm-datadog-repo/datadog",
-        release_name = "datadog",
-        flags = [
-            "--values",
-            chart_values,
-        ],
+        release_name = "datadog-agent-%s" % agent_image_tag,
```

Comment:
When `agent_image_tag` has periods in it like `"7.50.3"`, a bunch of errors come up during the helm install:

```
Running cmd: ['helm', 'upgrade', '--install', '--values', '/Users/noelle/dev/DirectEmployers/tilt-extensions/datadog/values.yaml', '--set', 'agents.image.tag=7.50.3', '--set', 'clusterAgent.image.tag=7.50.3', '--set', 'clusterChecksRunner.image.tag=7.50.3', 'datadog-agent-7.50.3', 'helm-datadog-repo/datadog']
     Release "datadog-agent-7.50.3" does not exist. Installing it now.
     W0209 07:35:46.202314   45402 warnings.go:70] metadata.name: this is used in Pod names and hostnames, which can result in surprising behavior; a DNS label is recommended: [must not contain dots]
     Error: 3 errors occurred:
     	* Service "datadog-agent-7.50.3" is invalid: metadata.name: Invalid value: "datadog-agent-7.50.3": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')
     	* Service "datadog-agent-7.50.3-cluster-agent" is invalid: metadata.name: Invalid value: "datadog-agent-7.50.3-cluster-agent": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')
     	* Service "datadog-agent-7.50.3-cluster-agent-admission-controller" is invalid: metadata.name: Invalid value: "datadog-agent-7.50.3-cluster-agent-admission-controller": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')
```

Reply:
Ah, I gotta do a string replacement for the release name :)  That's what I get for not testing it. :/

Reply:
Fixed and tested this time